readme.md

```diff
@@ -65,6 +65,19 @@ const el = (
 document.body.appendChild(el);
 ```
 
+### Alternative usage
+
+You can avoid configuring your JSX compiler by just letting it default to `React` and exporting the `React` object:
+
+```js
+const {React} = require('dom-chef');
+```
+
+This has the advantage of enabling `Fragment` support with the TypeScript compiler, if you're using it compile JSX without Babel. Related issue: https://github.com/Microsoft/TypeScript/issues/20469
+
+```
+TS17016: JSX fragment is not supported when using --jsxFactory
+```
```

Comment:
I added this to possibly reach people who google that and to make the exact error clear.